hedera-node/src/main/java/com/hedera/services/sigs/order/HederaSigningOrder.java

```diff
@@ -446,20 +446,24 @@ private boolean hasNondeprecatedAdminKey(ContractUpdateTransactionBody op) {
 			} else {
 				List<JKey> required = new ArrayList<>();
 				if (waclShouldSign) {
```

Comment:
These changes to `HederaSigningOrder` will conflict with refactoring I'm doing for https://github.com/hashgraph/hedera-services/issues/1890; I'm replacing the,
```
	final Predicate<TransactionBody> updateAccountSigns;
	final BiPredicate<TransactionBody, HederaFunctionality> targetWaclSigns;
```
fields with a dedicated `SignatureWaivers` class. I'd suggest reverting the changes to this file, if you don't mind. 🙏

Reply:
Not at all. Reverted all changes to file.